src/demo-app/app/demo-app/demo-app.css

```diff
@@ -192,3 +192,30 @@ md-toolbar .md-toolbar-layout md-toolbar-row {
 .demo-content .small-demo:first-child {
   margin-top: 40px;
 }
+
+.clazz-all {
+  font-style: italic;
+}
+
+.clazz-xs {
```

Comment:
I understand this is a convention with it's roots in Java but it's one of those conventions I wish wouldn't have existed ( developers & naming ).

_Perhaps_
`fx-class` && `fxClass` for selectors / variables respectivly ?

This is personal preference coming from a Java heavy shop, completely up to @ThomasBurleson